WC_Layers/TwoLayers_Geometry.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Oct 11 15:56:57 2024
+
+@author: Anupama Rajendra
+"""
+
+import openmc
+
+# Create geometry
+#Spherical shell:
+def make_spherical_shells(layers, inner_radius):    
+    '''
+    Creates a set of concentric spherical shells, each with its own material & inner/outer radius.
+    
+    layers: list of tuples with OpenMC Material name and thickness: (material, thickness)
+    inner_radius: the radius of the innermost spherical shell
+    '''

```

Comment:
I wonder if this function could take materials and thicknesses separately, and make layers within the function. This would avoid having the user provide list of tuples beforehand

```suggestion
def make_spherical_shells(materials, thicknesses, inner_radius):    
    '''
    Creates a set of concentric spherical shells, each with its own material & inner/outer radius.
    
    layers: list of tuples with OpenMC Material name and thickness: (material, thickness)
    inner_radius: the radius of the innermost spherical shell
    '''
    layers = zip(materials, thicknesses)
```